dockerfiles/base/scripts/base/commands/cmd_start.sh

```diff
@@ -45,12 +45,16 @@ cmd_start() {
   info "start" "Starting containers..."
   COMPOSE_UP_COMMAND="docker_compose --file=\"${REFERENCE_CONTAINER_COMPOSE_FILE}\" -p=\"${CHE_MINI_PRODUCT_NAME}\" up -d"
 
+  ## validate the compose file (quiet mode)
+  docker_compose --file=${REFERENCE_CONTAINER_COMPOSE_FILE} config -q || (error "Invalid docker compose file content at ${REFERENCE_CONTAINER_COMPOSE_FILE}" && return 2)
```

Comment:
maybe with only che.env it's not possible but a dev could break the docker-compose file